nucypher/policy/conditions/types.py

```diff
@@ -63,6 +64,7 @@ class ContractConditionDict(RPCConditionDict):
 #
 #
 class CompoundConditionDict(TypedDict):
```

Comment:
Probably missed by me, but this should be `CompoundConditionDict(_AccessControlCondition)`.